test/jdk/tools/jpackage/helpers/jdk/jpackage/test/AdditionalLauncher.java

```diff
@@ -81,6 +82,14 @@ public AdditionalLauncher addRawProperties(
         return this;
     }
 
+    public String getRawPropertyValue(String key) {
+        Map.Entry<String, String> entry = rawProperties.stream()
```

Comment:
This can be simply
`rawProperties.stream().findAny(item -> item.getKey().equals(key)).map(e -> e.getValue()).orElse(null);`

Or slightly better:
```
public String getRawPropertyValue(String key, Supplier<String> getDefault) {
    return rawProperties.stream().findAny(item -> item.getKey().equals(key)).map(e -> e.getValue()).orElseGet(getDefault);
}

```

Then we can create a function that will return the expected description of an additional launcher:
```
private String getDesciption(JPackageCommand cmd) {
    return getRawPropertyValue("description", () -> cmd.getArgumentValue("--description", unused -> "Unknown"));
}
```

This will let you avoid `if (expectedDescription != null)` checks and **always** verify launcher description.